topics/reference_guide/custom_language_support/additional_minor_features.md

```diff
@@ -173,6 +173,16 @@ EP: `com.intellij.listSplitJoinContext`
 needs to be implemented to define the exact behavior of splitting and joining list-like constructs
 in a language.
 
+### Rename and Change Signature Refactoring
+
+EP: `com.intellij.suggestedRefactoringSupport`
+
+[`SuggestedRefactoringSupport`](upsource:///platform/lang-api/src/com/intellij/refactoring/suggested/SuggestedRefactoringSupport.kt)
+provides functionality to implement rename and change signature refactorings for custom languages.
```

Comment:
the key is to automatically _suggest_ such refactorings by detecting such changes

Reply:
Fixed